bot/bot.go

```diff
@@ -0,0 +1,56 @@
+package bot
+
+import (
+	"fmt" //to print errors
+
+	"github.com/bwmarrin/discordgo"   //discordgo package from the repo of bwmarrin .
+	"github.com/qerdcv/muzlag/config" //importing our config package which we have created above
+)
+
+var BotId string
+var goBot *discordgo.Session
+
+func Start(cfg config.Config) {
```

Comment:
Consider to create Bot structure that will contain BotID, Session (discordgo.Session), and config.
Then create New function, that will return configured Structure, then Run method of Bot structure that will run bot